### ISSUE TITLE
Lack of options for benchmarking mode

### ISSUE THREAD

**Issue by Roaders** [ethereum-mining/ethminer#231]
On my system I have the following devices listed:

```
Listing OpenCL devices.
FORMAT: [deviceID] deviceName
[0] Intel(R) HD Graphics 630
        CL_DEVICE_TYPE: GPU
        CL_DEVICE_GLOBAL_MEM_SIZE: 13663212340
        CL_DEVICE_MAX_MEM_ALLOC_SIZE: 2147483647
        CL_DEVICE_MAX_WORK_GROUP_SIZE: 256
[1] GeForce GTX 1080 Ti
        CL_DEVICE_TYPE: GPU
        CL_DEVICE_GLOBAL_MEM_SIZE: 11811160064
        CL_DEVICE_MAX_MEM_ALLOC_SIZE: 2952790016
        CL_DEVICE_MAX_WORK_GROUP_SIZE: 1024


Listing CUDA devices.
FORMAT: [deviceID] deviceName
[0] GeForce GTX 1080 Ti
        Compute version: 6.1
        cudaDeviceProp::totalGlobalMem: 11811160064
```

but when I run a benchmark it uses the on-motherboard intel gpu and not the GeForce GTX. Obviously I want to benchmark the GeForce...

I have tried using the options available for the mining mode:

`ethminer -M -G --opencl --opencl-device 1`

but this seems to have no effect.

We need a lot of the same options for benchmarking as we do for mining...

**Comment by EoD** [comment]
@Roaders I think you misunderstand the parameters here.

First of all, the parameter ``-G`` and ``--opencl`` are identical. No need to specify them both.
You do not have two different devices, but two different platforms here. I guess you want to specify ``--opencl-platform 1`` instead of your device.

All in all: ``ethminer -G -M --opencl-platform 0`` and  ``ethminer -G -M --opencl-platform 1``

I do agree, that the ``--list-devices`` in this case is misleading and might need some improvements to show platform and devices properly. Let's rename the bug to "Improve --list-devices to show opencl platform".

**Comment by DeadManWalkingTO** [comment]
I think this issue can be closed.

**Comment by smurfy** [comment]
Fixed by #307
